src/nvim/globals.h

```diff
@@ -979,7 +979,9 @@ EXTERN char_u e_notopen[] INIT(= N_("E484: Can't open file %s"));
 EXTERN char_u e_notopen_2[] INIT(= N_("E484: Can't open file %s: %s"));
 EXTERN char_u e_notread[] INIT(= N_("E485: Can't read file %s"));
 EXTERN char_u e_nowrtmsg[] INIT(= N_(
-        "E37: No write since last change (add ! to override)"));
+    "E37: No write since last change (add ! to override)"));
+EXTERN char_u e_nowrtmsg_api[] INIT(= N_(
+    "E37: No write since last change (use force=true to override)"));
```

Comment:
I don't think this is a good idea. Maintaining different flavors of error messages for the API is extra work for little gain. Alternative is to document that the `force` parameter on the API method means "!" or bang (could rename the param as `bang` as well). It's completely reasonable to expect API users to understand this.

Also adding a `api` parameter and codepath to `ex_win_close` is similarly unpleasant.

Reply:
Ok, I will revert it.